packages/babel-plugin-transform-typescript/test/fixtures/optimize-const-enums/export-const-enum-type-no-deopt/stderr.txt

```diff
@@ -1,12 +1,12 @@
-The exported identifier "WhitespaceFlag" is not declared in Babel's scope tracker
+[<CWD>/packages/babel-plugin-transform-typescript/test/fixtures/optimize-const-enums/export-const-enum-type-no-deopt/input.ts] The exported identifier "WhitespaceFlag" is not declared in Babel's scope tracker
```

Comment:
Minor drive-by change because I was getting a warning from a test and I couldn't understand which test.